extensions/blocks/button/button-colors-panel.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	ContrastChecker,
+	PanelColorSettings,
+	__experimentalPanelColorGradientSettings as PanelColorGradientSettings,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { IS_GRADIENT_AVAILABLE } from './constants';
+
+export default function ButtonColorsPanel( {
+	backgroundColor,
+	fallbackBackgroundColor,
+	fallbackTextColor,
+	gradientValue,
+	setBackgroundColor,
+	setGradient,
+	setTextColor,
+	textColor,
+} ) {
+	const ButtonContrastChecker = (
+		<ContrastChecker
```

Comment:
This only seems to show when the colors are hovered. Once I move the mouse away it disappears...